zeppelin-integration/pom.xml

```diff
@@ -37,14 +37,9 @@
   </prerequisites>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I read https://maven.apache.org/plugins/maven-resources-plugin/examples/encoding.html and found we are using system-dependent encoding. Is it ok? Otherwise, is it better to set UTF-8 in a root pom? WDYT?

Reply:
You are right, we should add `project.build.sourceEncoding` in root pom.xml. Please keep in mind that this change also affects other plugins. For example the compiler plugin https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#encoding.